src/subscription.js

```diff
@@ -170,9 +167,6 @@ class Subscription extends EventEmitter {
   _whenReadyRejecter(err) {
     this.removeListener('ready', this._boundWhenReadyResolver);
     this._whenReadyRejectFn(err);
-    // Reset `whenReady` handlers
-    this._whenReadyResolveFn = null;
-    this._whenReadyRejectFn = null;
```

Comment:
Was there ever a purpose to resetting `_whenReadyRejectFn` to null? eg an `if (this._whenReadyRejectFn)`?

Reply:
I don't think there was. I saw that in the past we reset it inside `_reset` function. But because of that we had errors during reconnection. So I moved it to `ready` event handlers. I think the only place where we can leave it is `stop` function. There we know for sure that subscription is stopped and we unsubscribe from `ready` and `nosub` events.